packages/manager/apps/pci-ai-endpoints/src/hooks/token/useDatagridColumns.tsx

```diff
@@ -0,0 +1,142 @@
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { OsdsChip } from '@ovhcloud/ods-components/react';
+import {
+  DatagridColumn,
+  DataGridTextCell,
+  ActionMenu,
+  DateFormat,
+  useFormattedDate,
+} from '@ovh-ux/manager-react-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { ODS_CHIP_SIZE } from '@ovhcloud/ods-components';
+import { useQueries } from '@tanstack/react-query';
+import { useGetTokens } from '@/hooks/api/database/token/useToken.hook';
+import { getTokenQueryOptions } from '@/components/utils/getTokenQueries';
+import { TokenData } from '@/types/cloud/project/database/token';
+
+export type UseDatagridColumnsProps = {
+  projectId: string;
+  noLimitation: Date;
+  onUpdate: (token: TokenData) => void;
+  onDelete: (token: TokenData) => void;
+};
+
+export const FormattedDate = ({ date }: { date: Date }) => {
+  const formattedDate = useFormattedDate({
+    dateString: date.toISOString(),
+    format: DateFormat.display,
+  });
+  return <>{formattedDate}</>;
+};
+
+export const useDatagridColumns = ({
+  projectId,
+  noLimitation,
+  onUpdate,
+  onDelete,
+}: UseDatagridColumnsProps) => {
+  const { t } = useTranslation('token');
+
+  const { data: tokenNames, isLoading: isNamesLoading } = useGetTokens({
+    projectId,
+  });
+
+  const tokenNamesArray: string[] = tokenNames || [];
+
+  const tokenDetailsQueries = useQueries({
+    queries: tokenNamesArray.map((name) =>
+      getTokenQueryOptions(projectId, name),
+    ),
+  });
+
+  const tokenItems: TokenData[] = useMemo(
+    () =>
+      tokenDetailsQueries
+        .map((query) => query.data)
+        .filter((token): token is TokenData => token !== undefined),
+    [tokenDetailsQueries],
+  );
+
+  const isLoading = useMemo(
+    () =>
+      isNamesLoading || tokenDetailsQueries.some((query) => query.isLoading),
+    [isNamesLoading, tokenDetailsQueries],
+  );
+
+  const columns: DatagridColumn<TokenData>[] = [
+    {
+      id: 'name',
+      label: t('ai_endpoints_token_name'),
+      cell: (token: TokenData) => (
+        <DataGridTextCell>{token.name}</DataGridTextCell>
+      ),
+    },
+    {
+      id: 'description',
+      label: t('ai_endpoints_token_description'),
+      cell: (token: TokenData) => (
+        <DataGridTextCell>
+          {token.description === '' ? '' : token.description}
```

Comment:
I don't understand the goal of this condition, it seems that you can simply replace it with token.description